src/openforms/forms/migrations/0009_transfer_can_submit.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.25 on 2021-12-13 11:43
+
+from django.db import migrations
+
+from openforms.forms.constants import SubmissionAllowedChoices
+
+
+def forwards_func(apps, schema_editor):
+    Form = apps.get_model("forms", "Form")
+    forms = Form.objects.all()
+
+    for form in forms:
+        if form.can_submit:
+            form.submission_allowed = SubmissionAllowedChoices.yes
+        else:
+            form.submission_allowed = SubmissionAllowedChoices.no_with_overview
+    Form.objects.bulk_update(forms, ["submission_allowed"])
```

Comment:
FYI - this could be simplified to this:

```py
Form.objects.filter(can_submit=True).update(submission_allowed=SubmissionAllowedChoices.yes)
Form.objects.filter(can_submit=False).update(submission_allowed=SubmissionAllowedChoices.no_with_overview)
```

And more complicated single query (hurts readability):

```py
Form.objects.update(
    submission_allowed=Case(
        When(can_submit=True, then=Value(SubmissionAllowedChoices.yes)),
        When(can_submit=False, then=Value(SubmissionAllowedChoices.no_with_overview)),
    )
)
```
for completeness sake.

You can leave this as it is, it looks correct.

Reply:
Ohh good to know! :slightly_smiling_face: